src/libnetdata/functions_evloop/functions_evloop.c

```diff
@@ -60,13 +61,28 @@ struct functions_evloop_globals {
     struct rrd_functions_expectation *expectations;
 };
 
+static void rrd_functions_worker_cancleller(void *data) {
```

Comment:
Small typo on `rrd_functions_worker_cancleller`

Reply:
fixed it